test/dsl/dsl_basics.jl

```diff
@@ -110,16 +117,7 @@ let
     @test rn == rn2
 end
 
-@testset "make_reaction_system can be called from another module" begin
-    ex = quote
-        (Ka, Depot --> Central)
-        (CL / Vc, Central --> 0)
-    end
-    # Line number nodes aren't ignored so have to be manually removed
-    Base.remove_linenums!(ex)
-    @test eval(Catalyst.make_reaction_system(ex)) isa ReactionSystem
-end
-
```

Comment:
This one unfortunately is marked as deleted/added. I put into a `let end` block like our other tests. If there is some special reason to have it in this form, we can move it back.

Reply:
Seems it was also moved one step down.

Reply:
ok, manged to sort this ouit now, diff should look better.